packages/esm-service-queues-app/src/queue-table/queue-table-metrics.component.tsx

```diff
@@ -23,6 +23,12 @@ function QueueTableMetrics({ selectedQueue }: QueueTableMetricsProps) {
           <QueueTableMetricsCard queueUuid={selectedQueue.uuid} status={status.uuid} headerLabel={status.display} />
         );
       })}
+      <QueueTableMetricsCard
+        queueUuid={selectedQueue.uuid}
+        status={null}
+        headerLabel={t('averageWaitingTimeTitle', 'Average waiting time')}
+        value={!isNaN(averageWaitTime) ? averageWaitTime : 0}
+      />
```

Comment:
The metrics cards were intended to be out of scope for this ticket.  It is unclear to me what "Average Wait Time" means in a view that displays patients in multiple states.

Reply:
I actually went through the designs, but now that I think of it with your statement, it makes sense.
I'll remove this.
Thanks!